deploy/008_deploy_base_universal_receiver_delegate.ts

```diff
@@ -0,0 +1,27 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { ethers } from "hardhat";
+
+const deployBaseUniversalReceiverDelegate: DeployFunction = async ({
+  deployments,
+  getNamedAccounts,
+}: HardhatRuntimeEnvironment) => {
+  const { deploy } = deployments;
+  const { owner } = await getNamedAccounts();
+
+  const deployResult = await deploy("LSP1UniversalReceiverDelegateInit", {
+    from: owner,
+    log: true,
+    gasPrice: ethers.BigNumber.from("20000000000"), // in wei
```

Comment:
You can write it like this to make it easier to read:

```suggestion
    gasPrice: ethers.BigNumber.from("20_000_000_000"), // in wei
```